modules/button/css/lib/variables.scss

```diff
@@ -1,71 +1,88 @@
 $_wdc-btn-icons-path: $wdc-system-icons-path !default;
 
-$wdc-btn-icon-size: $wdc-spacing-s;
-$wdc-btn-icon-size-l: $wdc-spacing-m;
+$wdc-btn-font-weight: 500;
 $wdc-btn-border-size: 1px;
 
 $wdc-btn-height-l: $wdc-spacing-xl;
 $wdc-btn-height-m: $wdc-spacing-m;
 $wdc-btn-height-s: $wdc-spacing-s + ($wdc-btn-border-size * 2);
-$wdc-btn-height-icon: $wdc-spacing-l;
 
 $wdc-btn-min-width-l: 112px;
 $wdc-btn-max-width-l: 288px;
 $wdc-btn-min-width-m: 80px;
 $wdc-btn-max-width-m: 200px;
 $wdc-btn-min-width-s: 56px;
 $wdc-btn-max-width-s: 120px;
-$wdc-btn-width-icon: $wdc-spacing-l;
-$wdc-btn-width-icon-rounded: 56px;
 
 $wdc-btn-padding-l: 0 $wdc-spacing-l;
 $wdc-btn-padding-m: 0 $wdc-spacing-m;
 $wdc-btn-padding-s: 0 $wdc-spacing-xxs;
 $wdc-btn-padding-icon: $wdc-spacing-xxxs;
-$wdc-btn-padding-icon-rounded: (($wdc-btn-height-l - $wdc-btn-icon-size-l)/2) ((
-        $wdc-btn-width-icon-rounded - $wdc-btn-icon-size-l
-      )/2);
 
-$wdc-btn-border-radius: 40px;
-$wdc-btn-border-radius-icon: 3px;
+$wdc-btn-border-radius: $wdc-spacing-xl;
+$wdc-btn-border-radius-icon: $wdc-spacing-xxxs;
 
 $wdc-btn-bg: $wdc-color-soap-200;
 $wdc-btn-bg-hover: $wdc-color-soap-500;
 $wdc-btn-bg-active: $wdc-color-soap-200;
 $wdc-btn-bg-focus: $wdc-color-soap-200;
 $wdc-btn-bg-disabled: $wdc-color-french-vanilla-100;
-$wdc-btn-bg-icon-active: $wdc-color-blueberry-500;
-$wdc-btn-bg-icon-disabled: $wdc-color-french-vanilla-100;
 $wdc-btn-bg-primary: $wdc-color-cantaloupe-400;
 $wdc-btn-bg-primary-hover: $wdc-color-cantaloupe-500;
 $wdc-btn-bg-primary-disabled: $wdc-color-cantaloupe-200;
 $wdc-btn-bg-delete: $wdc-color-cinnamon-100;
 $wdc-btn-bg-delete-hover: $wdc-color-cinnamon-400;
 $wdc-btn-bg-delete-disabled: $wdc-color-cinnamon-100;
 
+$wdc-btn-border: $wdc-color-soap-500;
+$wdc-btn-border-hover: $wdc-color-licorice-100;
+$wdc-btn-border-active: $wdc-color-blueberry-400;
+$wdc-btn-border-focus: $wdc-color-blueberry-400;
+$wdc-btn-border-disabled: $wdc-color-soap-500;
+$wdc-btn-border-primary: $wdc-color-cantaloupe-400;
+$wdc-btn-border-primary-hover: $wdc-color-cantaloupe-400;
+$wdc-btn-border-primary-active: $wdc-color-blueberry-400;
+$wdc-btn-border-primary-disabled: $wdc-color-cantaloupe-400;
+$wdc-btn-border-delete: $wdc-color-cinnamon-500;
+$wdc-btn-border-delete-hover: $wdc-color-cinnamon-500;
+$wdc-btn-border-delete-active: $wdc-color-blueberry-400;
```

Comment:
Should be `$wdc-color-cinnamon-500`

Reply:
This is actually a bug in the React button. Do you mind updating it there as well?

Reply:
will do!